### ISSUE TITLE
[Feature Request/Suggestion] - Hiding unused randomized stuff from the Items Categories.

### ISSUE THREAD

**Issue by Ethryan** [jupiter0fire/OoTMMR_tracker_pack#62]
Hi, Just a thought.
How about hiding stuff that hasn't been checked, for example:
![image](https://github.com/user-attachments/assets/0766f5d6-c9ad-4948-8bb9-dc8815f6b730)
needs to be checked for:
![image](https://github.com/user-attachments/assets/98306a91-c524-4d04-badc-14cc48f5bd82)
to show up, this would reduce clutter in the Item tracker since Items not in use for that specific run just takes up space in the large list of items.

A really nice bonus would be if:
![image](https://github.com/user-attachments/assets/2100ec50-9dfa-47b6-9596-3fd277d69f70)
Removed the items from both Items OOT and Items MM: 
![image](https://github.com/user-attachments/assets/012a7897-9738-43ad-809a-929a00a7e605)
^-- List is cut off since I can have all of it shown on the screen at one time.
And added them to a new Items Shared category.
In order to reduce the amount of duplicates shown.

Anyways, its a really nice tracker in all other regards, thanks for making it.

**Comment by wbsch** [comment]
Hey, sorry for the late reply.

As far as we know, it's not actually possible to do this in EmoTracker without an insane amount of work. Hiding items would leave holes in the grid, and re-flowing is just not something EmoTracker item grids are meant to do. Again, this might just be us not knowing EmoTracker deeply enough - if you can come up with a low-effort way to do this that doesn't involve a lot of manual work each time the item grid sees changes in future, feel free to create a pull request!

**Comment by Agent-coulson** [comment]
"if you can come up with a low-effort way to do this that doesn't involve a lot of manual work each time the item grid sees changes in future"

Possibly look into the tracker for OoTMM by Ariah(on EmoTracker) for an example? idk what was involved there, but they do include this functionality. this functionality is also included on OoT trackers in EmoTracker too. don't play MM randos so no idea on that front.

**Comment by jupiter0fire** [comment]
Yes, that could be similarly implemented for our Emotracker pack. Unfortunately, there is no clean solution in Emotracker to properly represent all these changes and it would be a lot of work.
We currently do not have plans to implement it, as we plan to discontinue the Emotracker pack at some point.

**Comment by Agent-coulson** [comment]
If I may be so bold as to ask, why do you plan on discontinuing this pack? was my immediate go-to for OoTMM randos and seems like the only really good and easy one to get and work with

**Comment by jupiter0fire** [comment]
you can have a look at wbsch's post on the OoTMM-Discord in the #tracker-dev channel drom 2/7/26
